storage/include/libstqe.sh

```diff
@@ -51,33 +51,33 @@ function stqe_init_fwroot
     for cmd in python python3 python2; do
         $cmd -V > /dev/null 2>&1 && python=$cmd && break
     done
-    [[ -n $python ]] || rlSkip "python not found"
+    [[ -n $python ]] || cki_skip_task "python not found"
 
     typeset pip_cmd=$([[ $python == python3 ]] && echo pip3 || echo pip)
     if [[ $fwbranch != "master" ]]; then
         if [[ -n $STQE_STABLE_VERSION ]]; then
-            rlRun "git checkout $STQE_STABLE_VERSION" || \
-                rlAbort "fail to checkout $STQE_STABLE_VERSION"
+            cki_run_cmd_pos "git checkout $STQE_STABLE_VERSION" || \
+                cki_abort_task "fail to checkout $STQE_STABLE_VERSION"
         fi
         if [[ -n $LIBSAN_STABLE_VERSION ]]; then
-            rlRun "$pip_cmd install libsan==$LIBSAN_STABLE_VERSION" || \
-                rlAbort "fail to install libsan==$LIBSAN_STABLE_VERSION"
+            cki_run_cmd_pos "$pip_cmd install libsan==$LIBSAN_STABLE_VERSION" || \
+                cki_abort_task "fail to install libsan==$LIBSAN_STABLE_VERSION"
         fi
     fi
 
     # install required packages
-    rlRun "bash env_setup.sh" $BKRM_RC_ANY
+    cki_run_cmd_neu "bash env_setup.sh"
 
-    rlRun "$python setup.py install --prefix=" || \
-        rlAbort "fail to install test framework"
+    cki_run_cmd_pos "$python setup.py install --prefix=" || \
+        cki_abort_task "fail to install test framework"
 
-    popd
+    cki_pd
 
     return 0
 }
 
 function stqe_fini_fwroot
 {
     typeset fwroot=$(stqe_get_fwroot)
-    rlRun "rm -rf $fwroot"
+    cki_run_cmd_neu "rm -rf $fwroot"
```

Comment:
Another one that should probably be `_pos`?

Reply:
The same reason to L38.